py-wacz/wacz/waczindexer.py

```diff
@@ -213,21 +214,15 @@ def generate_metadata(self, res, wacz):
 
         desc = res.desc or self.desc
         title = res.title or self.title
-        textIndex = PAGE_INDEX if res.text else ''
         data = {}
         if title:
             package_dict['title'] = title
 
         if desc:
             package_dict['desc'] = desc
-
-        if textIndex:
-            package_dict['textIndex'] = textIndex
-
-        package_dict['pages'] = [
-            {'title': page.get('title') or page.get('url'),
-             'date': timestamp_to_iso_date(page['timestamp']),
-             'url': page['url']} for page in self.pages.values()]
+    
+        if res.text:
+            package_dict['textIndex'] = PAGE_INDEX
```

Comment:
No longer need this in the package_dict, since can just look for `pages/pages.jsonl`, which is already included in the manifest.

Instead, can add a `mainPageURL` and `mainPageTS` for the main page URL and TS, if specified via --url flag. (If not specified, the TS is determined automatically from the first match of the main URL)